Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM registry.access.redhat.com/ubi7/ubi:7.7 as anchore-cli-builder
+FROM registry.access.redhat.com/ubi8/ubi:8.1 as anchore-cli-builder
 
 ######## This is stage1 where anchore wheels, binary deps, and any items from the source tree get staged to /build_output ########
 
-ENV LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8
+ENV LANG=en_US.UTF-8 LC_ALL=C.UTF-8
```

Comment:
I'm curious about this one, was `en_US` causing issues?

Reply:
ubi8 (and I believe it is becoming more common for other 'min' style distros) are not including full suite of locales by default - in the spirit of keeping the dependencies limited, i've switched to the default (available) C.UTF-8 which I believe should be sufficient for the CLI - some related discussions/similiar info here:


https://github.com/tianon/docker-brew-debian/issues/45